### PR TITLE
Make python code in Sequential docs consistent

### PR DESCRIPTION
**Pull request by 01-vyom**
As discussed in issue #14971, 

Removed `>>>` to make the python code example consistent.

### CODE REVIEW

File: keras/engine/sequential.py [hunk 1]
```diff
@@ -46,44 +46,44 @@ class Sequential(functional.Functional):
...
+  model.add(tf.keras.layers.Dense(4))
+  # model.weights not created yet
+
+  # Whereas if you specify the input shape, the model gets built
+  # continuously as you are adding layers:
+  model = tf.keras.Sequential()
+  model.add(tf.keras.layers.Dense(8, input_shape=(16,)))
+  model.add(tf.keras.layers.Dense(4))
+  len(model.weights)
   4
```
Comment (fchollet) [inline keras/engine/sequential.py:75]: IMO the whole thing does become a lot more readable as code blocks like this. But the output of the code ("4" here) should be moved to a comment, like `# Returns "4"`

File: keras/engine/sequential.py [hunk 2]
```diff
@@ -46,44 +46,44 @@ class Sequential(functional.Functional):
...
-  >>> len(model.weights)
+  # When using the delayed-build pattern (no input shape specified), you can
+  # choose to manually build your model by calling
+  # `build(batch_input_shape)`:
+  model = tf.keras.Sequential()
+  model.add(tf.keras.layers.Dense(8))
+  model.add(tf.keras.layers.Dense(4))
+  model.build((None, 16))
+  len(model.weights)
   4
```
Comment (fchollet) [inline keras/engine/sequential.py:85]: Here too, etc.

File: keras/engine/sequential.py (hunk 1 above)
Comment (01-vyom) [reply keras/engine/sequential.py:75]: Yup that change would be good. Also, there are lots of codes where these `>>>` exists so should I make changes for them all one by one?

**Review by fchollet — APPROVED** [review]
LGTM. Note that such changes should only be done in other files if we're in a similar situation where we have inconsistent formatting or where we just have very long code comments in the examples.

**Pull request merged**